tap_watir.gemspec

```diff
@@ -35,6 +35,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 1.16"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", "~> 3.0"
+  spec.add_development_dependency "rubocop" "~> 0.58.1"
```

Comment:
let's use `spec.add_development_dependency "rubocop", "~> 0.50"`
the `~>` will allow anything that is < 1.0 and Selenium is requiring `~> 0.50.0` which means anything that is < 0.51.0
Also don't forget the comma in there.